github-electron/github-electron.d.ts

```diff
@@ -740,7 +751,7 @@ declare namespace Electron {
 	 * The BrowserWindow class gives you ability to create a browser window.
 	 * You can also create a window without chrome by using Frameless Window API.
 	 */
-	class BrowserWindow extends EventEmitter {
+	class BrowserWindow extends EventEmitter implements Destroyable {
```

Comment:
Although it is not a modification in this PR, I think we can use `NodeJS.EventEmitter`.  It was interface previously, but now it's a class and can be extended.